Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -129,4 +142,229 @@ private function getItemsPerPage(ResourceInterface $resource, Request $request)
 
         return $resource->getItemsPerPageByDefault();
     }
+
+    /**
+     * Determines whether output walkers should be used.
+     *
+     * @param QueryBuilder $queryBuilder
+     *
+     * @return bool
+     */
+    private function useOutputWalkers(QueryBuilder $queryBuilder)
```

Comment:
@dunglas I too don't get why this should be in another place, when it's specific to Doctrine pagination. But then again I'm not sure I can understand very well the other strong opinions of @sroze